stable/pomerium/README.md

```diff
@@ -47,48 +58,63 @@ The command removes nearly all the Kubernetes components associated with the cha
+| `ingress.enabled`                       | Enables Ingress for pomerium                                                                                                                                                                               | `false`                                                                            |
+| `ingress.annotations`                   | Ingress annotations                                                                                                                                                                                        | `{}`                                                                               |
+| `ingress.hosts`                         | Ingress accepted hostnames                                                                                                                                                                                 | `nil`                                                                              |
+| `ingress.tls`                           | Ingress TLS configuration                                                                                                                                                                                  | `[]`                                                                               |
+| `metrics.enabled`                       | Enable prometheus metrics endpoint                                                                                                                                                                         | `false`                                                                            |
+| `metrics.port`                          | Prometheus metrics endpoint port                                                                                                                                                                           | `9090`                                                                             |
+## Upgrading
+
+### 2.0.0
+
+- The authenticate service is broken into two different Service objects.  One is meant for end users to interact with, and the other is for internal GRPC based communication.
+  - You must generate an SSL cert that is valid for 
+    - `[authenticate.name].[rootDomain]` | default: `authenticate.[domain]`
+    - `[authenticate.internalName].[namespace].svc.cluster.internal` | default: `authenticate-internal.default.svc.cluster.local`
+    - `[authenticate.name].[namespace].svc.cluster.internal` | default: `authenticate.default.svc.cluster.local`
+  - You may need to run `helm upgrade --force` to recreate the authorize service correctly
```

Comment:
only the authorize service or did you mean authenticate?

Reply:
Only authorize should have a change to the ClusterIP (or at least, that's the intention).  Authenticate will get a new headless service, so it won't be impacted.

Reply:
yes, right, sorry, didn't think enough about it :)